src/routers/messages.py

```
@@ -55,32 +71,37 @@ async def enqueue(payload: dict | list[dict]):
                     for p in payload
                 ]
 
-                print("Number of queue records to insert:", len(queue_records))
+                logger.debug(f"Inserting {len(queue_records)} queue records")
 
                 # Use insert to maintain order
                 stmt = insert(QueueItem).returning(QueueItem)
                 result = await db.execute(stmt, queue_records)
```

Ruff (0.8.2)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)